src/contracts/core/AVSDirectoryStorage.sol

```diff
@@ -64,5 +92,5 @@ abstract contract AVSDirectoryStorage is IAVSDirectory {
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[43] private __gap;
+    uint256[35] private __gap;
```

Comment:
looks like gap was decremented incorrectly? Have to check original `feat/operator-set-release` branch as well